scripts/cli_create_kb.py

```diff
@@ -0,0 +1,43 @@
+"""Functionality for creating the knowledge base from downloaded assets and by querying Wikipedia's API."""
+import logging
+import os
+from pathlib import Path
+
+import spacy
+import typer
+
+from extraction import get_paths
+from kb import WikiKB
+
+
+def main(vectors_model: str, language: str):
+    """Create the Knowledge Base in spaCy and write it to file.
+    language (str): Language.
+    vectors_model (str): Name of model with word vectors to use.
+    """
+    logger = logging.getLogger(__name__)
+    logger.info("Constructing knowledge base.")
+
+    nlp = spacy.load(vectors_model, exclude=["tagger", "lemmatizer", "attribute_ruler"])
+    kb = WikiKB(
+        nlp.vocab,
+        nlp(".").vector.shape[0],
```

Comment:
Is there a more elegant way to reliably get embedding vector length for any model?

Reply:
I don't actually think so, unless I'm missing it too. `doc.vector` can be sourced from a user hook, from `doc.tensor` or from the tokens sourcing from `vocab.vectors`. If you don't know all the details from your pipeline up-front, this way might be the easiest to check.